README.md

```diff
@@ -242,7 +242,7 @@ const mySchema = new Schema({
 ### ES Modules
 
 Nano ID provides ES modules. You do not need to do anything to use Nano ID
-as ESM in webpack, Parcel, or Node.js.
+as ESM in Webpack, Parcel, or Node.js.
```

Comment:
Are you sure that webpack team use capitalizated name?

Reply:
Actually, no. This is a slip, let it stay lowercase